packages/components/src/guide/stories/index.js

```diff
@@ -17,20 +12,19 @@ import Guide from '../';
 export default {
 	title: 'Components/Guide',
 	component: Guide,
-	parameters: {
-		knobs: { disable: false },
+	argTypes: {
+		contentLabel: { control: 'text' },
+		finishButtonText: { control: 'text' },
+		onFinish: { action: 'onFinish' },
 	},
 };
 
-const ModalExample = ( { numberOfPages, ...props } ) => {
+const Template = ( { onFinish, ...props } ) => {
 	const [ isOpen, setOpen ] = useState( false );
 
 	const openGuide = () => setOpen( true );
 	const closeGuide = () => setOpen( false );
 
-	const loremIpsum =
-		'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
-
```

Comment:
No strong opinion, but I removed this lorem ipsum for simplicity.

By the way the CSS in the modal seems pretty broken, but this is an existing problem.

Reply:
> By the way the CSS in the modal seems pretty broken, but this is an existing problem.

Absolutely, In fact, in the current example on trunk, the dots are not click-able (while they are in this PR).

Overall the component is not looking in its best shape either